script.js

```diff
@@ -1,6 +1,275 @@
-console.log("hello script js");
+//Initialize Values
+var seatsSoldKL = 0;
+var economySoldBL = 0;
+var businessSoldBL = 0;
+var firstSoldBL = 0;
+var priceKL = 50;
+var econPrice = 50;
+var bizPrice = 50;
+var firstPrice = 50;
+var location;
+var cabinClass;
 
-var inputHappened = function(currentInput){
-  console.log( currentInput );
-  return "WOW SOMETHING HAPPENED";
-};
+//Starting prompt message
+document.querySelector("#output").innerText =
+`Welcome! Where will your next adventure take you? Type selection:
+-KL
+-Bali`;
+
+
+
+// Calculations
+
+
+function priceCategory(seatsSold, numberOfSeats) {
+  switch (true) {
+    case (seatsSold === 1):
+    return `There are ${1 - seatsSold} seats left for this price.`;
+
+    case (seatsSold < Math.ceil(numberOfSeats / 2)):
+    return `There are ${Math.ceil(numberOfSeats / 2) - seatsSold} seats left for this price.`;
+
+    case (seatsSold < numberOfSeats):
+    return  `There are ${numberOfSeats - 1 - seatsSold} seats left for this price.`;
+
+    case  (seatsSold === numberOfSeats):
+    return  `There are ${numberOfSeats - seatsSold} seats left for this price.`;
+
+  }
+}
+
+function calculateKLPrice(seatsSold) {
+  if (seatsSold > 0) {
+    if (seatsSold < 5) {
+      priceKL *= 1.03;
+    } else if (seatsSold === 9) {
+      priceKL = 91000;
+    } else {
+      priceKL *= 1.05;
+    }
+  }
+}
+
+function calculateEconPrice(seatsSold) {
+  if (seatsSold > 0) {
+    if (seatsSold < 8) {
+      econPrice *= 1.03;
+    } else if (seatsSold === 14) {
+      econPrice = 91000;
+    } else {
+      econPrice *= 1.05;
+    }
+  }
+}
+
+function calculateBizPrice(seatsSold) {
+  if (seatsSold > 0) {
+    if (seatsSold < 3) {
+      bizPrice *= 1.06;
+    } else if (seatsSold === 5) {
+      bizPrice  = 91000;
+    } else {
+      bizPrice *= 1.1 ;
+    }
+  }
+}
+
+function calculateFirstPrice(seatsSold) {
+  if (seatsSold > 0) {
+    if (seatsSold === 3) {
+      firstPrice = 191000;
+    } else {
+      firstPrice *= 1.15;
+    }
+  }
+}
+
+
+
+// Prompt sequence
+//If location has not been specified yet (i.e. this is the user's first input)
+var inputHappened = function(input){
+  if (!location) {
+    //Output depending on kl, bali, or invalid input
+    if (input.toLowerCase() === "kl") {
+      location = input.toLowerCase();
+      document.querySelector("#input").value = "";
+      return `Purchase a ticket? Type selection:
```

Comment:
This line stops the function from going further, so the subsequent codes will not be triggered :)